src/core/Expander/Expander/Expander.test.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { axeTest } from '../../../utils/test/axe';
+import { Expander, ExpanderProps } from './Expander';
+import {
+  ExpanderContent,
+  ExpanderTitleButton,
+  ExpanderTitleButtonProps,
+} from '../';
+
+describe('Basic expander', () => {
+  const TestExpanderWithProps = (titleProps: ExpanderTitleButtonProps) => {
+    return (
+      <Expander className="expander-test">
+        <ExpanderTitleButton
+          {...titleProps}
+          {...{ 'data-testid': 'expander-title' }}
+        />
+        <ExpanderContent>Test expander content</ExpanderContent>
+      </Expander>
+    );
+  };
+
+  const TestExpander = TestExpanderWithProps({
+    ariaCloseText: 'click to close expander',
+    ariaOpenText: 'click to open expander',
+    children: 'Test expander',
+  });
+
+  it('render with the same component on the same container does not remount', () => {
+    const { getByTestId, rerender } = render(TestExpander);
```

Comment:
Is this `TestExpander` really useful as it just used once? 
Could it be directly in the render as done with the rerender?